provisioning/roles/jenkins/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Ensure dependencies are installed.
+  apt: pkg=curl state=installed
+
+- name: Add Jenkins apt repository key.
+  apt_key:
+    url: "http://pkg.jenkins-ci.org/debian/jenkins-ci.org.key"
+    state: present
+
+- name: Add Jenkins apt repository.
+  apt_repository:
+    repo: "deb http://pkg.jenkins-ci.org/debian binary/"
+    state: present
+    update_cache: yes
+
+- name: Ensure Jenkins is installed.
+  apt: pkg=jenkins state=installed
+
+# Make sure Jenkins starts, then configure Jenkins.
+- name: Ensure Jenkins is started and runs on startup.
+  service: name=jenkins state=started enabled=yes
```

Comment:
If you can setup everything while jenkins is stopped, great, it not, at the end of the tasks, stop jenkins